wls-common/src/main/java/org/jboss/arquillian/container/wls/CommonWebLogicConfiguration.java

```diff
@@ -527,4 +529,15 @@ public boolean isDeployExplodedArchive() {
    public void setDeployExplodedArchive(boolean deployExplodedArchive) {
        this.deployExplodedArchive = deployExplodedArchive;
    }
+
+   /**
+    * Remote machine configuration flag. Specify whether the target server is located on another machine (ie. if the
+    * target server do not share the same file system as the test server).
+    *
+    * @return {@code true} if executing test on a remote machine, {@code false} otherwise.
+    */
+   // Note: getter-only, setter must be implemented in child classes.
+   public boolean isRemoteMachine() {
+      return remoteMachine;

```

Comment:
Could we not assume remoteMachine when adminUrl != localhost|12.7.0.0.1 ? 

What harm does it do to use remote even if it's not 100% remote, e.g. some other local ip interface used then 127.0.0.1?